Docs/source/usage/faq.rst

```diff
@@ -27,6 +27,17 @@ We change the default in ``cmake/dependencies/AMReX.cmake``.
 Note that the tiny profiler adds literally no overhead to the simulation runtime, thus we enable it by default.
 
 
+What do I need to know about using the boosted frame?
+-----------------------------------------------------
+The boost transformations are calculated internally to WarpX, so the input deck can be designed in the lab frame.  Little modification is needed beyond setting ``warpx.gamma_boost``.
+There are new instabilities that arise (see :ref:`boosted frame theory <theory-boostedframe>`) that require different algorithms for stability; please see the :ref:`boosted examples <usage-examples>` for more help.
+That said, there are some practical items worth considering that we list here to assist in designing and analyzing boosted frame simulations:
+
+- The boosted frame simulation begins at boosted time :math:`t'=0`.
+- Best practice is to set the upper :math:`z` limit of the problem domain to 0 with ``geometry.prob_hi``.
+Otherwise it is possible that the boosted simulation begins with downstream objects unexpectedly in the problem domain before there is a chance to correctly calculate all interactions.
+
```

Comment:
Let's add a link to the in-depth section on boosted frame: https://warpx.readthedocs.io/en/latest/theory/boosted_frame.html aka ref anchor `theory-boostedframe`

Reply:
done